hyperspy/api_nogui.py

```diff
@@ -3,17 +3,17 @@
 # Set the PyQt API to 2 to avoid incompatibilities between matplotlib
 # traitsui
 
+from hyperspy import docstrings
```

Comment:
Revert this changes: the logger needs to be imported and called before the rest.